data_structures/lowest_common_ancestor_test.py

```diff
@@ -0,0 +1,41 @@
+# pytest for lowest common ancestor in a binary tree
+
+# External
+import pytest
+
+# Internal
+from lowest_common_ancestor import BinaryTree, find_ancestor
+
+bt = BinaryTree([10, 6, 4, 5, 3, 7, 8, 9, 15, 14, 13, 16])
+root = bt.root
+n1_1 = 3  # test 1
+n2_1 = 5
+r_1 = 4
+n1_2 = 9  # test 2
+n2_2 = 5
+r_2 = 6
+n1_3 = 6  # test 3
+n2_3 = 13
+r_3 = 10
+n1_4 = 8  # test 4
+n2_4 = 9
+r_4 = 8
+n1_5 = 13  # test 5
+n2_5 = 16
+r_5 = 15
```

Comment:
Just put this in the list below. What is the point of making them individual variables?

Reply:
changes implemented